### DISCUSSION TITLE
Cruiser 4.1.0 (Beta)

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#544] [Announcements]
## Cruiser 4.1.0 (Beta) is [available](https://www.dropbox.com/scl/fo/0pi1svwz14rl49l2gldqr/ANVb1vCX3ezM-cuXr532fFY?rlkey=3lt3t50tinxdsdkfz0py1t4ec&st=e2snb8iu&dl=0):

- https://github.com/devemux86/cruiser/discussions/538

<img src="https://github.com/devemux86/cruiser/assets/3484020/a5554eab-228d-4bf3-ab61-330ffa590fbf" width="50%"/>

**Comment by Leeb64** [comment]
Unfortunately, the nogo Aerias disappear when the route is deleted.
The next time you plan your route you have to enter the areas again...
Suboptimal.

**Reply by devemux86** [reply]
[BRouter website](https://brouter.de/brouter-web/) has a separate option for no-go areas in the clear dialog.
We could think of something like that.

![BRouter](https://github.com/devemux86/cruiser/assets/3484020/bb2c72cd-a5ea-4813-b082-f6eebb76ad54)

**Reply by Leeb64** [reply]
Would be more practical!
The environmental zones or other unattractive zones don't just disappear...
If you could save these nogo Aerias under favorites (to be on the safe side)

**Reply by devemux86** [reply]
> If you could save these nogo Aerias under favorites

This is another idea, but it requires much more work. :slightly_smiling_face:

**Reply by WolfgangZ42** [reply]
We had already a possibility to store "permanent no-go-areas" - see https://github.com/devemux86/cruiser/discussions/538#discussioncomment-9443944

**Comment by Leeb64** [comment]
Saving it would also make sense because every now and then some “things” disappear when the app is updated.
Recently some of my saved routes disappeared.
A backup copy always makes sense.

**Reply by devemux86** [reply]
No-go areas are saved / loaded / transferred with the routes.

**Reply by Leeb64** [reply]
If the restricted areas only have to be entered for the one route to be planned, I can actually enter a detour straight away using a shaping point or two... the work is pretty much the same.
Restricted areas only make sense if they are there permanently.

**Reply by devemux86** [reply]
It is not the same and with waypoints it is much more work, e.g. to avoid a city:
- With waypoints, we must find the roads around the city and place waypoints there.
And if those roads are under the avoid options? Then need to make more changes.

- With no-go areas, just place a no-go circle above the city and all done quickly in 1 step.

**Reply by WolfgangZ42** [reply]
That's right. And therefore the implementation of route-related no-go-areas is important! 

But don't forget the permanent no-go-areas - imagine a road which is closed for the next months ...

**Reply by devemux86** [reply]
Probably as easily accessible and placed no-go areas?
Like the favorites for home, work, etc.

**Reply by WolfgangZ42** [reply]
For my opinion it needs not to be easily accessible - just in the Cruiser directory of my SDcard. 
Of course you have to know how to create that .gpx-files.

**Reply by devemux86** [reply]
> Of course you have to know how to create that .gpx-files.

There is not any designer to create automatically such external files,
unless we use a desktop editor, something most users will never do.

See how simple the routing websites offer no-go areas: [BRouter](https://brouter.de/brouter-web/), [OpenRouteService](https://maps.openrouteservice.org/), [Valhalla](https://valhalla.openstreetmap.de/).

Note on mobile we do not have a desktop mouse to design complicated shapes.

**Reply by MotoUKRider** [reply]
This is a nice feature, it worked well on the few tries I made - I can see both views on whether they should be "per route" or "global" as you may wish to make a minor detour on a route without affecting the current "road avoidances", but also may wish to keep an area active whilst there are road-works etc.

I half-expected online BRouter to offer a polygon/polyfile facility, that would certainly allow for accurate tailoring of the no-go area.  How you would achieve that in-app without the mouse is another story... the gestures can probably be captured.

**Comment by devemux86** [comment]
Cruiser 4.1.0-b2 is available:
- Support BRouter external no-go files with internal no-go areas

**Reply by DolomitesTom** [reply]
Is there a Dokumentation how to define No Go areas via Bouter? Will it be possible in the future to define no-go areas in cruiser and export as GPX file in order to open as external nogo area?

**Reply by devemux86** [reply]
Information about BRouter no-go files exists here:
- https://github.com/abrensch/brouter/blob/master/docs/users/android_advanced.md#using-nogo-areas
- https://brouter.de/brouter/vianogo.html

Export no-go areas in Cruiser, we can also see it in the implementation of favorite no-go areas:
- https://github.com/devemux86/cruiser/discussions/545

**Reply by devemux86** [reply]
@DolomitesTom For example see the 2 BRouter no-go gpx by @WolfgangZ42 here:
- https://github.com/devemux86/cruiser/discussions/538#discussioncomment-9524093

**Reply by DolomitesTom** [reply]
Thanks

**Comment by bios4** [comment]
> @bios4 thanks again for continuing to suggest improvements to the app texts! 👍

Here we go again for some suggestions for the German translations...
This time about the naming of the POI types in the search field:

<img src="https://github.com/devemux86/cruiser/assets/145460147/65a9903b-4b7d-4db1-b282-9aa8058aee72" width="33%">

1. Rename to "Essen & Trinken" ("Speisen" looks too formal); alternatively rename it to just "Restaurant"
2. Rename to "Kaffeehaus"
3. Rename to "Bar"
4. Rename to "Supermarkt" ("Markt" means mainly something like a farmer's market)
5. Rename to "Einkaufen"

**Reply by Leeb64** [reply]
2. "Kaffeehaus" sounds a bit old fashioned ...I think "Café"  is better.

**Reply by Leeb64** [reply]
Regarding 5: “Einkaufen” is also too general.
“Geschäft” might be better...(German word for Shop).

**Reply by bios4** [reply]
> Regarding 5: “Einkaufen” is also too general.
“Geschäft” might be better...(German word for Shop).

While I agree on the "Kaffeehaus" <> "Café" wording 😉  I really dislike "Geschäft" and don't find that "Einkaufen" is too general ("to shop" == "einkaufen", too)...

But it's as always: ask five persons, get six opinions. 😆

**Reply by Leeb64** [reply]
Complaining on a high level...there are more important things...

**Reply by devemux86** [reply]
Sorry for the late reply, too much work on Cruiser lately. :slightly_smiling_face: 

- (**1**) contains: restaurant, fast_food, biergarten

- (**4**) contains: convenience, supermarket, bakery

**Comment by DolomitesTom** [comment]
The link for 4.1.0 beta does not show any 4.1.x version for downloading but only 5.0.x

**Reply by devemux86** [reply]
Cruiser 4.1 (Beta) was replaced with Cruiser 5 (Beta).

(that's why this topic is now closed)

**Reply by DolomitesTom** [reply]
how can I get the latest 4.1? I am still at 4.0.2. And will Cruiser 5 get the possibility to rotate the map in planning mode. I found this very important.

**Reply by devemux86** [reply]
@DolomitesTom 

Already answered [above](https://github.com/devemux86/cruiser/discussions/544#discussioncomment-10241702), there is no more Cruiser 4.1 (Beta).

Either advance to Cruiser 5 or stay with Cruiser 4.
